### ISSUE TITLE
EncryptedLocalStore  ERROR

### ISSUE THREAD

**Issue by yuanyunhua** [airsdk/Adobe-Runtime-Support#3322]
When I updated the version to 51 and packaged the Apple app, EncryptedLocalStore reported an error and returned version 50.2 as normal. I feel like reading and writing multiple ELS files at the same time will crash.

**Comment by yuanyunhua** [comment]
There were no issues with local testing, but there were issues when users downloaded and used it after it was packaged and launched.

**Comment by itlancer** [comment]
May be it's all around AIR 51 changes for ELS. Related issue: https://github.com/airsdk/Adobe-Runtime-Support/issues/3283

**Comment by ajwfrost** [comment]
Yes I think we have some related problems; just checking this one so will try to get an update on it soon; there's some fallback that also should happen which may explain some of the quirks here..

**Comment by funyamora** [comment]
I am always grateful for your help.
A bug in EncryptedLocalStore is causing a serious problem, so could you please fix it as soon as possible?

**Comment by itlancer** [comment]
For me all these issues fixed with latest AIR 51.1.1.5.
